### PR TITLE
Additional configuration options for workspacer.Bar

### PR DESCRIPTION
**Pull request by N1x0**
This PR aims to introduce more configuration options for the `BarPlugin`, to allow changing the Bar's width and position as a fix to #212 
Adds the following features:

* `bool BarIsTop` sets the position of the bar
  * Added in:  102139f2bf6bc9ef8a63f98ea3cf823c846851ea
  * Default is `true`

* `bool BarIsMinimal` sets the bar width to the width of all widgets 
  * Added in:  468751354694da71eb895198dad0d64f833e7730
  * Default is `false`

* `bool BarReservesSpace` sets the offset in the `MenubarLayoutEngine`
  * Added in: bb8db8575fa33163dbb8d89686dc8aa760cae568
  * Default is `true`

Tasks
 - [ ] dynamically adjust barwidth at runtime to avoid painting over other widgets.
 - [ ]  Force the bar  to stay on top when no space is being reserved for it
 - [ ]  Move the bar in bottom position when the windows taskbar is auto-hiding

### CODE REVIEW

**Comment by dalyIsaac** [comment]
This is great!

There's an issue with multiple monitors. The bar doesn't appear in the correct location for monitor 1 - currently it displays at the red line:

image

If needed I can debug this tomorrow.

File: src/workspacer.Bar/BarForm.cs [hunk 1]
```diff
@@ -66,11 +65,17 @@ private void OnLoad(object sender, EventArgs e)
         {
             this.Height = _config.BarHeight;
             var titleBarHeight = this.ClientRectangle.Height - this.Height;
-            this.Location = new Point(_monitor.X, _monitor.Y - titleBarHeight);
+           
+            if (_config.BarIsTop)
+                this.Location = new Point(_monitor.X, _monitor.Y - titleBarHeight);
+            else
+
+                this.Location = new Point(_monitor.X, _monitor.Height - _config.BarHeight);
```
Comment (dalyIsaac) [inline src/workspacer.Bar/BarForm.cs:73]:
```suggestion
                this.Location = new Point(_monitor.X, _monitor.Y + _monitor.Height - _config.BarHeight);
```
This resolved the issue on my setup.

**Comment by azinsharaf** [comment]
Hi @josteink  Do you have a rough time estimate when this PR would be available? I really like the idea of having the bar in the bottom. Eyes can catch is faster (at least for me)

**Comment by josteink** [comment]
Why ask me? I'm not working on it?

@N1x0 created the PR. @dalyIsaac provided feedback. Since then no new work has been done. 

Ask the 2 parties actually involved 😅

**Comment by azinsharaf** [comment]
thanks @josteink 

@N1x0 and @dalyIsaac , Do you have a rough time estimate when this PR would be available? I really like the idea of having the bar in the bottom. Eyes can catch is faster (at least for me)

**Comment by josteink** [comment]
Closed due to inactivity.

**Pull request closed without merging**